app/serializers/npq_application_csv_serializer.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require "csv"
+
+class NPQApplicationCsvSerializer
+  attr_reader :scope
+
+  def self.headers
+    %w[
+      id
+      participant_id
+      full_name
+      email
+      email_validated
+      teacher_reference_number
+      teacher_reference_number_validated
+      school_urn
+      headteacher_status
+      eligible_for_funding
+      funding_choice
+      course_id
+      course_name
+    ]
+  end
```

Comment:
Can we call them `csv_headers`? For a second there I thought they are response headers.

Reply:
I will rename to `csv_headers` and make it a private method